v2/lua/openresty_org/controller.lua

```diff
@@ -18,6 +18,19 @@ local format = string.format
 local unescape_uri = ngx.unescape_uri
 local match_table = {}
 
+local repo_file = {}
+repo_file[1] = [[[openresty]
+name=Official OpenResty Repository
+baseurl=https://]]
+repo_file[2] = ""
+repo_file[3] = ""
+repo_file[4] = [[-$basearch/
+skip_if_unavailable=True
+gpgcheck=1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/openresty/openresty/pubkey.gpg
```

Comment:
We'd better use `https://openresty.org/yum/openresty/openresty/pubkey.gpg` here for `/cn/` users as well.